src/guides/v2.3/extension-dev-guide/cache/partial-caching/create-cache-type.md

```diff
@@ -83,3 +83,51 @@ You must specify the following parameters:
 
 [tagscope]: {{ site.mage2bloburl }}/{{ page.guide_version }}/lib/internal/Magento/Framework/Cache/Frontend/Decorator/TagScope.php
 [type]: {{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Customer/Model/Cache/Type/Notification.php
+
+## Examples {#m2devgde-cache-type-model}
+
+A cache type `translate` is declared in the Magento_Translation module using the `cache.xml` configuration file.
+
+```xml
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Cache/etc/cache.xsd">
+    <type name="translate" translate="label,description" instance="Magento\Framework\App\Cache\Type\Translate">
+        <label>Translations</label>
+        <description>Translation files</description>
+    </type>
+</config>
+```
+
+Translate cache type model class is defined in `Magento\Framework\App\Cache\Type\Translate.php` class. It must extends `Magento\Framework\Cache\Frontend\Decorator\TagScope` class.
```

Comment:
```suggestion
Translate cache type model class is defined in `Magento\Framework\App\Cache\Type\Translate.php` class. It must extend the `Magento\Framework\Cache\Frontend\Decorator\TagScope` class.
```

Reply:
Thanks for the review @BarnyShergold  I have committed the changes requested.